tests/ui_tests/test_cluster_list_page.py

```diff
@@ -638,3 +641,86 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
             assert (
                 cluster_components_page.check_that_save_btn_disabled()
             ), "Save button should be disabled"
+
+
+class TestClusterStatusPage:
+    @pytest.mark.usefixtures("create_community_cluster")
+    def test_open_by_tab_cluster_status_page(self, app_fs):
+        cluster_main_page = ClusterMainPage(app_fs.driver, app_fs.adcm.url, 1).open()
```

Comment:
probably you should replace `1` with `create_community_cluster.id`

Reply:
replaced everywhere